examples/cfrl_adult.ipynb

```diff
@@ -0,0 +1,1449 @@
+{
```

Comment:
<p>Please import explainers from <code>alibi.explainers</code>  as that is the public package</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>